services/workflows-service/prisma/migrations/20250202144546_added_document_file_cascade/migration.sql

```diff
@@ -0,0 +1,11 @@
+-- DropForeignKey
+ALTER TABLE "DocumentFile" DROP CONSTRAINT "DocumentFile_documentId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "DocumentFile" DROP CONSTRAINT "DocumentFile_fileId_fkey";
```

Comment:
Before final commit i suggest to delete migraiton file and create a new single migration file so it will easier to follow

Reply:
Will do it before merging the epic branch. 👍🏼